src/DynamoCoreWpf/Views/Core/NodeView.xaml

```diff
@@ -78,9 +78,14 @@
                               CommandParameter="CrossProduct"
                               Header="{x:Static p:Resources.ContextMenuLacingCrossProduct}" />
                 </MenuItem>
+                <MenuItem Name="nodeIsFrozen"
+                          Header="{x:Static p:Resources.NodesRunStatus}"
+                          IsChecked="{Binding IsExplicitFrozen,Mode=OneWay}"   
+                          IsEnabled="{Binding CanToggleFrozen, Mode=OneWay}"
+                          Command="{Binding Path=ComputeRunStateOfTheNodeCommand}" />
```

Comment:
This command can be removed if we're not currently doing workspace or application level freeze stuff.